services/cd-service/pkg/repository/transformer.go

```diff
@@ -763,12 +763,18 @@ func (c *CreateEnvironmentApplicationLock) Transform(ctx context.Context, state
 }
 
 type DeleteEnvironmentApplicationLock struct {
+	Authentication
 	Environment string
 	Application string
 	LockId      string
 }
 
 func (c *DeleteEnvironmentApplicationLock) Transform(ctx context.Context, state *State) (string, error) {
+	// TODO (BB): add the "*" check on the checkUserPermissions() method.
```

Comment:
handle todo

Reply:
Already handled and removed the comment.